libgalois/include/katana/GraphTopology.h

```diff
@@ -409,7 +409,8 @@ class KATANA_EXPORT ShuffleTopology : public EdgeShuffleTopology {
         },
         katana::no_stats());
 
-    KATANA_LOG_DEBUG_ASSERT(std::is_sorted(degrees.begin(), degrees.end()));
+    KATANA_LOG_DEBUG_ASSERT(
+        std::is_sorted(degrees.begin(), degrees.end(), std::greater<>()));
```

Comment:
You can add a helpful comment that the check on is_sorted should match the sorting order of degrees (point to the function where sorting happens).